lms/static/sass/shared/_header.scss

```diff
@@ -765,7 +765,7 @@ header.global-new {
       }
     }
 
-    @include media( $desktop ) {
+    @include media( $large ) {
       .wrapper-header {
         width: 100%;
         min-width: 800px;
```

Comment:
Not your doing, but could use some clean up (in a separate project/PR by the org).